modules/api-support-tiers.adoc

```diff
@@ -29,4 +29,4 @@ Components and developer tools that receive continuous updates through the Opera
 == API tier 4
 No compatibility is provided. API and AOE can change at any point. These capabilities should not be used by applications needing long-term support.
 
-It is common practice for Operators to use custom resource definitions (CRDs) internally to accomplish a task.  These objects are not meant for use by actors external to the Operator and are intended to be hidden.  If any CRD is not meant for use by actors external to the Operator, the `operators.operatorframework.io/internal-objects` annotation in the Operators `ClusterServiceVersion` (CSV) must be specified, and that signals that the corresponding resource is internal use only.
+It is common practice for Operators to use custom resource definitions (CRDs) internally to accomplish a task.  These objects are not meant for use by actors external to the Operator and are intended to be hidden.  If any CRD is not meant for use by actors external to the Operator, the `operators.operatorframework.io/internal-objects` annotation in the Operators `ClusterServiceVersion` (CSV) should be specified to signal that the corresponding resource is internal use only and the CRD may be explicitly labeled as tier 4.
```

Comment:
There are two spaces after 'hidden.' and before 'If.' 

There are two spaces after 'task.' and before 'These'.

Reply:
fixed